test/suite/test_tiered19.py

```diff
@@ -116,6 +116,26 @@ def test_gcp_filesystem(self):
         self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
             lambda: ss.ss_customize_file_system(
                 session, no_access_bucket, None, self.get_fs_config(prefix)), err_msg)
+        
+         # We cannot use the file system to create files, it is readonly.
+        # So use python I/O to build up the file.
+        f = open('foobar', 'wb')
+
+        outbytes = ('MORE THAN ENOUGH DATA\n'*100000).encode()
+        f.write(outbytes)
+        f.close()
```

Comment:
Instead of calling `close` here we can do 
```
with open('file_name', ''wb) as f:
    outbytes = ('MORE THAN ENOUGH DATA\n'*100000).encode()
    f.write(outbytes)
```

This will automatically close the file handle for us when we exit the `with` block.